envoy/server/overload/overload_manager.h

```diff
@@ -99,6 +99,13 @@ class OverloadManager : public LoadShedPointProvider {
    * Get a factory for constructing scaled timer managers that respond to overload state.
    */
   virtual Event::ScaledRangeTimerManagerFactory scaledTimerFactory() PURE;
+
+  /**
+  // Stop the overload manager timer and wait for any pending resource updates to complete.
```

Comment:
nit: it seems like the comments in this file use the `*` instead of `//`, so would be good to change to `*` for consistency